src/reconciler/exec/external.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: MIT
+use vstd::{prelude::*, view::*};
+
+verus! {
+
+pub trait ExternalLibrary<InputType: View, OutputType: View> {
```

Comment:
I think this is used when the controller does not need to communicate with any external systems? Could we have some doc (comments) here?